modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/jaxrs/application/AccountAdminApplication.java

```diff
@@ -90,6 +93,11 @@ else if (!accountEntryEmailAddressValidator.isValidDomain(
 
 			errorMessage = "x-has-an-invalid-email-domain";
 		}
+		else if (accountEntryEmailAddressValidator.isEmailAddressAlreadyExist(
```

Comment:
I don't like the name but I cannot image something different. For sure it should be isEmailAddressAlreadyExist**s** like in other places (try search AlreadyExist in the code)

Reply:
Can you propose me an alternative? :)

Reply:
What about isEmailAddressAlreadyUsed?